systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java

```diff
@@ -63,12 +70,63 @@
 @Tag(REGRESSION)
 @Tag(BRIDGE)
 @Tag(INTERNAL_CLIENTS_USED)
+@SuiteDoc(
+    description = @Desc("Test suite for various Kafka Bridge operations."),
+    contact = @Contact(name = "Lukas Kral", email = "lukywill16@gmail.com"),
```

Comment:
I don't think this makes sense. You do not want to have tests tied to a particular person.

Reply:
So having a `@Contact` does not make sense for you right? Do you mean remove it completely or maybe re-place with multiple responsible person?

Reply:
It is a community, people come and go, and so on. So if someone has any questions, they should come to GitHub Discussions, Slack, or Mailing list. Not to the contact person. If you need some tracking, I would maybe consider having something like `@Authors`. But I would try it as unrelated to any particular person(s) as possible. I would probably also use only GitHub handles or something rather than email.

But as it is right now - with `@Contact` and `email`, it is just as if you are telling someone who tries to run it to send Lukas an email with some questions.

Reply:
I can imagine having there handle for the "strimzi-systemtest-contributors" group, but that doesn't need to be in every test.
Having something like `@Authors` will be fine, but IMO it will anyway be taken as list of people which should be pinged because the test is failing (so similarly to the `@Contact`)

Reply:
> Having something like @Authors will be fine, but IMO it will anyway be taken as list of people which should be pinged because the test is failing (so similarly to the @Contact)

So should eliminate the `@Contact` and not use it at all? It seems to be the best option I assume...Option with having an `@Authors` annotation will be very redundant...and I am not sure we want to have all names in all tests but maybe I don't know.... What is your thoughts about this @jankalinic @jankalinic @Frawless ? If there are no any....I would proceed with not using `@Contact`

Reply:
If current implementation allow you to remove, feel free to remove it, I am fine with it. The field is used for reporting purposes and atm we don't need it here. In case `Contact` is required, you will have to update it in the generator itself.

Reply:
To me it seems authors or contact is sufficient on it's own. Having strimzi st contributors there instead would be great though.